Monika After Story/game/cardgames.rpy

```diff
@@ -0,0 +1,4781 @@
+                m "You were really trying this time!"
+                m "Great job, [player]!"
+
+            else:
+                m "And you won! Nice~"
+
+    elif store.mas_nou.game.total_turns < 25:
+        if store.mas_nou.player_win_streak > 0:
+            $ choice = renpy.random.randint(0, 2)
+
+            if not choice:
+                m "Another quick win for you!"
+
+                if renpy.random.randint(0, 4) > 3:
+                    m "But you better not to relax, [player]~"
+
+            elif choice == 1:
+                m "Wow, [player]!"
+                # TODO: this doesn't make sense imo
+                m "I can't keep up with you!"
+
+            else:
+                if len(store.mas_nou.game.monika.hand) > 3:
+                    m "Maybe I should try a bit harder?~"
+                    m "Ehehe, you keep finishing [_round!t]s before I can do anything."
+
+                else:
+                    m "Ah...{w=0.1}I was so close!"
+                    m "Good job, [player]!"
+
+        elif (
+            len(store.mas_nou.game.monika.hand) > 4
+            or not store.mas_nou.game.player.yelled_nou
+        ):
+            m "Wow!{w=0.2} Played all your cards already?"
+            m "That was quick!"
+
+        else:
+            $ choice = renpy.random.randint(0, 2)
+
+            if not choice:
+                m "Well played!"
+
+            elif choice == 1:
+                m "Impressive, [player]!"
+
+            else:
+                m "That was a quick [_round!t] for you!"
+
+    elif store.mas_nou.game.total_turns > 75:
+        $ choice = renpy.random.randint(0, 2)
+
+        if not choice:
+            m "A quite long [_round!t], [player]."
+
+            if len(store.mas_nou.game.monika.hand) < 4:
+                if store.mas_nou.player_win_streak > 0:
+                    m "And I almost win this time!"
+
+                else:
+                    m "And I almost win!"
+
+                m "Ahaha~ Well played!"
+
+            else:
+                m "Well played!"
+
+        elif choice == 1:
+            m "That was intense!"
+
+        else:
+            m "Ehehe~ You're really good!"
+
+    else:
+        $ choice = renpy.random.randint(0, 3)
+
+        if not choice:
+            if store.mas_nou.player_win_streak > 0:
+                m "You won again!"
+
+            else:
+                m "You won!~"
+
+        elif choice == 1:
+            m "This [_round!t] is yours!"
+
+        elif choice == 2:
+            m "And you won! Good job!"
+            if renpy.random.randint(0, 8) > 7:
+                m "But don't expect to win everytime~"
+
+        else:
+            if store.mas_nou.monika_win_streak > 1:
+                m "I'm glad you won this time~"
+
+            m "Good job, [player]!"
+    return
+
+label mas_nou_reaction_player_wins_game:
+    $ choice = renpy.random.randint(0, 3)
+
+    if not choice:
+        m "Oh!{w=0.2} Actually you won this game!"
```

Comment:
should be "Oh!{w=0.2} You actually won this game!"

Reply:
Are you sure? Because with that it sounds rude to me. Like "oh so you're capable to win this".
What if we placed a comma after actually?

Reply:
What I mean is that the actually at the start just seems off/wrong to me, sounds weird, even with a comma. That being said, I understand your view about sounding rude. I'll defer to you if you wanna adjust or not, but my updated recommendation that tries to keep the uniqueness of these lines. "Oh!{w=0.2} I didn't notice you were so close to victory." (new line) "Good job on winning, ehehe~"